scripts/apps/monitoring/MultiActionBarReact.tsx

```diff
@@ -133,22 +133,75 @@ export class MultiActionBarReact extends React.Component<IProps, IState> {
                 </div>
             );
         } else {
+            const groups = groupBy(this.state.actions, 'group');
+            const groupNames = Object.keys(groups);
+
             return (
                 <div data-test-id="multi-actions-inline">
                     {
-                        this.state.actions.map((action, i) => (
-                            <button
-                                onClick={() => {
-                                    this.onTrigger(action);
+                        groupNames.map((group, i) => (
+                            group === 'undefined') ?
+                            groups[group].map((action, key) => (
+                                <button
+                                    onClick={() => {
+                                        this.onTrigger(action);
+                                    }}
+                                    className="navbtn strict"
+                                    title={action.label}
+                                    key={key}
+                                    data-test-id={action.label}
+                                >
+                                    <Icon className={action.icon} size={22} />
+                                </button>
+                            )) :
+                            <DropdownTree
+                                className="dropdown navbtn strict"
```

Comment:
I wonder if that className + different styles could be named somehow so it would describe what it does..
maybe some `inline={true}` property could trigger that class and style rather than this

Reply:
yea, I'm not happy with this neither. Issue is that DropdownTree has some default inline styles that are overriding styles in classes, because of that I needed to add both things (className and stylings). This case is especially for nav btns, and can be applied only in this multiedit bar. We had meeting today and we agreed on creating some unified dropdown component that can cover all the cases.

Reply:
ok so can you add some comment that this will be handled?

Reply:
Hmm, but yes, I think we can go with inline, I just checked other places where this is implemented, it will work in any case